epoxy-sample/src/main/java/com/airbnb/epoxy/VerticalGridCardSpacingDecoration.java

```diff
@@ -45,6 +45,9 @@ public void getItemOffsets(Rect outRect, View view, RecyclerView parent, State s
       // Only item in row
       outRect.left = outerPadding;
       outRect.right = outerPadding;
+      // Assign top and bottom padding if there is just one item in the row
+      outRect.top = innerPadding;
```

Comment:
@elihart So in the current sample app, all the buttons don't have a vertical padding between them. I thought it would be better of symmetrical reasons to provide a default padding between all items (no matter the span count) and then if someone wants to increase it, they can do so via the xml